packages/go/openapi/doc/openapi.json

```diff
@@ -605,7 +605,7 @@
                   "type": "object",
                   "properties": {
                     "data": {
-                      "$ref": "#/components/schemas/model.auth-provider"
+                      "$ref": "#/components/schemas/model.oidc-provider"
```

Comment:
the json is generated from the yaml and i'm not seeing any yaml file changes, did you update the json file directly?